### PR TITLE
ECS

### PR DESCRIPTION
**Pull request by dominikwilkowski**
Working to get the latest changes from floem: https://github.com/lapce/floem/pull/432

### CODE REVIEW

File: src/ui/app_view.rs
Comment (thewizzy) [inline src/ui/app_view.rs:361]: Should remove the untrack from here, it stops the main window from changing

Comment (dominikwilkowski) [reply src/ui/app_view.rs:361]: You are of course very right! The `untrack` makes me nervous for all those side effects further down the line. From the discussions in discord it sounds like there will be a `dyn_container` soon that has the old functionality without subscribing signals automatically. So maybe we wait until that work is merged in before merging this PR?

**Review by thewizzy — APPROVED** [review]

**Pull request merged**
